src/package.json

```diff
@@ -262,6 +262,11 @@
 				"key": "ctrl+shift+k",
 				"mac": "cmd+shift+k",
 				"when": "true"
+			},
+			{
+				"command": "editor.action.inlineSuggest.commit",
+				"key": "tab",
+				"when": "inlineSuggestionVisible && editorTextFocus && !editorTabMovesFocus && !inSnippetMode && !suggestWidgetVisible && editorLangId != 'markdown'"
```

Comment:
I'm not sure whether this is the best solution, this is what sonnet thought up for enabling me to accept suggestions on empty lines with tab. I was kind of surprised tab didn't work on lines without anything on it tbh.

Reply:
What does this do? What if multiple extensions try to do this?